static/templates/pod.html

```diff
@@ -0,0 +1,12 @@
+<div class="panel panel-default">
```

Comment:
Note I've nicked the styling here for the details panel above it, which I was updating anyway in https://github.com/rapidpro/casepro/pull/110